scripts/cleanup_old_vms.py

```diff
@@ -125,6 +145,16 @@ def process_provider_vms(provider_key, provider_type, matchers,
                 else:
                     vm_creation_time = provider.vm_creation_time(vm_name)
 
+                if vm_creation_time is False:
+                    # This VM must have some problem, include in report even though we can't delete
+                    status = provider.vm_status(vm_name)
+                    deleted_vms_list.append([provider_key,
+                                             vm_name,
+                                             NULL,  # can't know age, failed getting creation date
+                                             status,
+                                             FAIL])
+                    raise Exception  # the except block message is accurate in this case
```

Comment:
@izapolsk we'll have to chat about this tomorrow because looking closer I'm not quite sure what you mean.